include/deal.II/fe/fe_tools_extrapolate.templates.h

```diff
@@ -1451,8 +1508,16 @@ namespace FETools
           queue.pop();
         }
 
-
-      u2.compress(VectorOperation::insert);
+      // Correctly synchronise values at ghost interface for all processors. For
```

Comment:
Not your fault -- we have a habit of using American English spelling:
```suggestion
      // Correctly synchronize values at ghost interface for all processors. For
```